projects/plugins/boost/app/assets/src/js/utils/make-admin-ajax-request.ts

```diff
@@ -22,7 +22,7 @@ export async function makeAdminAjaxRequest< T = JSONObject >( payload: JSONObjec
 		method: 'post',
 		body: new URLSearchParams( {
 			...payload,
-			...{ nonce: Jetpack_Boost.criticalCssAjaxNonce },
+			...{},
```

Comment:
I had to add an empty object because of typescript errors. There are two fixes to this issue.
* Either we should remove type of payload `JSONObject`  from  ```export async function makeAdminAjaxRequest< T = JSONObject >( payload )``` function parameters.
* Pass an empty object.

NOTE: can't use ```// @ts-ignore``` it will throw some build errors. Build gets fail.

Reply:
May I suggest this instead:
```
body: new URLSearchParams( payload as Record<string, string> ),
```

Reply:
@eAdnan007 thank you so much ❤️  for the heads up. I was trying something similar but at the wrong place. I have fixed it. Thanks once again, it was great learning. ❤️

Reply:
🤔 

If we're jamming the payload into a `Record<string, string>`, isn't it better to make the payload argument of the `makeAdminAjaxRequest` be a `Record< string, string >`?

i.e.:
```
export async function makeAdminAjaxRequest< T = JSONObject >( payload: Record< string, string > ): Promise< T > {
```

That way, anyone who calls this method will be required to marshall their data into string/string key/value pairs themselves.

Reply:
@thingalon, thank you for the suggestion. Now, the code looks cleaner as compared to the previous attempts. 😄